assignments/prototypes.js

```diff
@@ -15,14 +15,27 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
-
+function GameObject(attribute) {
+  this.createdAt = attribute.createdAt;
+  this.name = attribute.name;
+  this.dimensions = attribute.dimensions;
+  this.destroy = function () {
+    return `${this.name} was removed from the game.`;
+  };
+}
```

Comment:
This game destroy function is not set up properly.  Here is the right way:  

function GameObject(properties) {
  this.createdAt = properties.createdAt;
  this.dimensions = properties.dimensions;
}

GameObject.prototype.destroy = function() {
  return `${this.name} was removed from the game.`
}